pkg/virt-controller/services/template.go

```diff
@@ -1500,6 +1500,10 @@ func (t *TemplateService) VMIResourcePredicates(vmi *v1.VirtualMachineInstance,
 	return VMIResourcePredicates{
 		vmi: vmi,
 		resourceRules: []VMIResourceRule{
+			// Run overcommit first to avoid overcommitting overhead memory
+			NewVMIResourceRule(func(vmi *v1.VirtualMachineInstance) bool {
+				return t.clusterConfig.GetMemoryOvercommit() != 100
```

Comment:
This might technically be problematic. If I hard coded "100" as my desired value, this function would decide to use the cluster default, when I really wanted no overcommit?